src/examples/mfem_mortar_lm_patch.cpp

```diff
@@ -0,0 +1,293 @@
+// Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and
+// other Tribol Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (MIT)
+
+#include <set>
+
+#ifdef TRIBOL_USE_UMPIRE
+// Umpire includes
+#include "umpire/ResourceManager.hpp"
+#endif
+
+// MFEM includes
+#include "mfem.hpp"
+
+// Axom includes
+#include "axom/CLI11.hpp"
+#include "axom/slic.hpp"
+
+// Redecomp includes
+#include "redecomp/redecomp.hpp"
+
+// Tribol includes
+#include "tribol/common/Parameters.hpp"
+#include "tribol/config.hpp"
+#include "tribol/interface/tribol.hpp"
+
+int main( int argc, char** argv )
+{
+  // initialize MPI
+  MPI_Init( &argc, &argv );
+  int np, rank;
+  MPI_Comm_size(MPI_COMM_WORLD, &np);
+  MPI_Comm_rank(MPI_COMM_WORLD, &rank);
+
+#ifdef TRIBOL_USE_UMPIRE
+  umpire::ResourceManager::getInstance();  // initialize umpire's ResouceManager
+#endif
+
+  // initialize logger
+  axom::slic::SimpleLogger logger;
+  axom::slic::setIsRoot(rank == 0);
+
+  // command line options
+  // number of times to uniformly refine the serial mesh before constructing the
+  // parallel mesh
+  int ref_levels = 2;
+  // polynomial order of the finite element discretization
+  int order = 1;
+
+  axom::CLI::App app { "mfem_mortar_lm_patch" };
+  app.add_option("-r,--refine", ref_levels,
+    "Number of times to refine the mesh uniformly.")
+    ->capture_default_str();
+  app.add_option("-o,--order", order, 
```

Comment:
Do we want to restrict this example using mortar + LM to order = 1?

Reply:
Commented out this option for now.  Will enable it again once LOR is working for implicit contact.